src/sentry/api/helpers/group_index/update.py

```diff
@@ -673,7 +680,7 @@ def process_group_resolution(
 def merge_groups(
     group_list: Sequence[Group],
     project_lookup: Mapping[int, Project],
-    acting_user: User,
+    acting_user: Any,
```

Comment:
changing all of these to `Any` doesn't seem right -- if it is not know what the correct type is it's better to leave the type off here (such that it can get filled in later when the file is upgraded to stricter typing)